packages/financial-health-calculator/src/calculator/acceptance-of-funding.ts

```diff
@@ -0,0 +1,27 @@
+import type { ScoreCalculationResult } from './../types/score/score-calculation-result';
+
+export const calculateAcceptanceOfFundingScore = (
+  packageName: string,
+  npmFundingUrl: string,
+  githubFundingUrl: string,
+): ScoreCalculationResult => {
+  const weight = 0.6;
```

Comment:
Can we refactor these weights to be values of a constants object in the constants file?

```js
const WEIGHTS = {
  LICENSE_TYPE: 0.4,
  ORGANISATIONAL_ACTIVITY: 0.2,
  // ...
}
```

Reply:
Resolved in 46a93d61fa8534f67d9b794fc06bd5620ec9225f